data/modules/Scout/ScanDisplay.lua

```diff
@@ -224,7 +224,7 @@ function scanDisplay:drawScanProgress(scan)
 	-- suitable across any style changes in the future.
 	local progressBarColor = ui.getStyleColor("SliderGrabActive")
 	ui.withStyleColors({ ["PlotHistogram"] = progressBarColor }, function()
-		ui.progressBar(completion, Vector2(width, 0), "Scan Progress")
+		ui.progressBar(completion, Vector2(width, 0), ls.MISSION_PROGRESS)
```

Comment:
I recommend displaying something like "Data Collected:" instead of "Mission Progress:" - it's a little bit clearer what's going on, and we don't have to change it later if a mission wants more than one body to be scanned at once (or e.g. an orbital scan and a surface scan of a specific area).

Reply:
I think there'd need to be some rework required regardless if we end up with missions which require multiple scans.

I've played with the different percentages again in the progress bar - showing the "coverage" has the same issue as the original percentage display, although we can alleviate it a little bit by increasing the number of decimals. Showing the "0..100%" is, imho, reverting back to being confusing.

Finally, as the label, "Data Collected" has no other reference; how about "Coverage" for now - this would be directly related to the tooltip "Target Coverage". But it would likely need to be reworked once there are missions with multiple steps.

**Data Collected with coverage percentage**
![image](https://github.com/user-attachments/assets/361b3ee0-73f8-4701-a31f-fc756b97ee02)

**Data Collected with progressbar percentage**
![image](https://github.com/user-attachments/assets/7db391f0-47ec-4779-83ab-fb4df4e33ef0)

**Coverage with coverage percentage**
![image](https://github.com/user-attachments/assets/d037630d-6b0b-4a03-b7eb-5953691c9fff)

**Coverage Progress with no percent**
![image](https://github.com/user-attachments/assets/f7a5ce01-992b-42a8-808b-1d87a2fe6a81)

**Coverage tooltip**
![image](https://github.com/user-attachments/assets/5aaa0edd-0e55-4494-ba7c-41562b5d8068)

----

What would be nice is an animated progress bar which animates only while the scan is active and data is being collected. That would be real and immediate feedback to the user and is something they should be familiar with. That would require significant work though as we'd need to modify the `ImGUI::ProgressBar` implementation.

Reply:
"Data" is referred to by almost all of the mission texts / BBS adverts. I think it meets the minimum threshold to not be confusing to the player (your scan generates data -> the mission giver wants scan data).

If you think "Data Collected: XX.X%" is confusing, then please just use the label "Data Collected" for the progress bar - personally I think it's clear enough that this is a separate element from the actual scan parameters which are displayed on the card. Adding an additional `ui.spacing()` before the scan progress bar might prevent confusion as well.

Reply:
Ok; I was thinking in the context of the scan, not the overall mission/game.

Based on the mockups above, which percentage do you think should be used here - the percentage of the progress-bar (0..100), or the percentage of the "target coverage" (0..[target coverage])?

(And thank you for your feedback!)

Reply:
Here's the latest version :

![Scan - new detail display](https://github.com/user-attachments/assets/a32cf75a-f834-4a5c-aba3-aaf986d9c836)

If approved, I'll squash the various commits.

Reply:
Progress bar should display progress bar completion, 0..100%. Top right of the active scan card should always display current coverage (in percent or km), reaching the target coverage value at Data Collected: 100%. (i.e. revert 975d7923db574e34fb94ff222247fea1720f319f).

Reply:
Thank you for proposing multiple different implementations and sticking with the PR! My goal here is to converge on the solution that provides the most relevant information to the player. I understand everyone has their own opinion and don't want anyone to feel like their ideas are discarded, but as the reviewer I'm trying to avoid a "too many cooks" situation.

Reply:
> Progress bar should display progress bar completion, 0..100%. Top right of the active scan card should always display current coverage (in percent or km), reaching the target coverage value at Data Collected: 100%. (i.e. revert [975d792](https://github.com/pioneerspacesim/pioneer/commit/975d7923db574e34fb94ff222247fea1720f319f)).

:+1: 

The original idea for adding the progress bar was to _replace_ the top-right percentage to reduce clutter, I had just not gotten around to that in order to first see whether or not the progress bar was feasible and useful.

> Thank you for proposing multiple different implementations and sticking with the PR! My goal here is to converge on the solution that provides the most relevant information to the player. I understand everyone has their own opinion and don't want anyone to feel like their ideas are discarded, but as the reviewer I'm trying to avoid a "too many cooks" situation.

Not at all! Thank you so much for your feedback. I'm worried I'm taking up too much of you guys' time with reviewing my silly little PR's..

Reply:
![image](https://github.com/user-attachments/assets/3b4545ad-e604-4f7f-be90-d92cdb720e7c)

Reply:
Let's give this a spin and then maybe remove the top right % in a follow up?

Reply:
> Let's give this a spin and then maybe remove the top right % in a follow up?

I had removed it (see screenshot above with all the red annotations) but @Web-eWorks requested it be added again...